eva-accession-import-dbsnp2/src/main/java/uk/ac/ebi/eva/accession/dbsnp2/io/DbsnpJsonClusteredVariantsWriter.java

```diff
@@ -51,16 +49,18 @@ public DbsnpJsonClusteredVariantsWriter(MongoTemplate mongoTemplate, ImportCount
     @Override
     public void write(List<? extends DbsnpClusteredVariantEntity> clusteredVariants) {
         try {
-            dbsnpClusteredVariantWriter.write(clusteredVariants);
+            if (!clusteredVariants.isEmpty()) {
+                dbsnpClusteredVariantWriter.write(clusteredVariants);
+            }
         } catch (BulkOperationException exception) {
-            List<String> ids = extractUniqueHashesForDuplicateKeyError(exception).collect(toList());
-            Query query = new Query();
-            query.addCriteria(Criteria.where("_id").in(ids));
-            List<DbsnpClusteredVariantEntity> entities = mongoTemplate.find(query, DbsnpClusteredVariantEntity.class);
-            List<Long> rsIDs = entities.stream()
-                .map(DbsnpClusteredVariantEntity::getAccession)
-                .collect(toList());
-            logger.error("Duplicate RS IDs: {}", rsIDs);
+            List<String> hashes = extractUniqueHashesForDuplicateKeyError(exception).collect(toList());
+            List<String> variantsThatFailedInsert =
```

Comment:
This is really small formatting thing in case you want to adopt it (I think looks better this way)
```
            List<String> variantsThatFailedInsert = clusteredVariants.stream()
                                                                     .filter(v -> hashes.contains(v.getHashedMessage()))
                                                                     .map(v -> v.getAccession().toString())
                                                                     .collect(toList());
```